frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -22,11 +24,31 @@ spec:
       - {key: environment, operator: In, values: [dev]}
 `);
 
+const pvcPhase = pvc => _.get(pvc, 'status.phase');
 const menuActions = [Cog.factory.ModifyLabels, Cog.factory.ModifyAnnotations, Cog.factory.Edit, Cog.factory.Delete];
 
+const PVCStatus = ({pvc}) => {
+  const phase = pvcPhase(pvc);
+  if (!phase) {
+    return '-';
+  }
+
+  switch (phase) {
+    case 'Pending':
```

Comment:
Could we Enum these phases ?

Reply:
Can't add enums in jsx :/ Needs TypeScript

Reply:
Ohh.... :(